cli.js

```diff
@@ -2,120 +2,74 @@
 import c from 'chalk';
 import program from 'commander';
 import fs from 'fs';
-import readline from 'readline';
 import util from 'util';
 
 import helper from './dgraphHelper';
 
-const rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout,
-});
-
-const get_schema = async () => {
-  const CLIENT = helper.create_client();
-  const fetched_schema = await helper.get_schema(CLIENT);
-  console.log(util.inspect(fetched_schema, false, null, true));
-}
-
-const get_diff = async client => helper.diff_checker(client);
+const { DB_URL: HOST } = process.env;
 
 const print_diff = differences => {
-  const CONFLICTS = differences[0];
-  const ADDED = differences[1];
+  const conflicts = differences[0];
+  const added = differences[1];
 
-  if (CONFLICTS.length > 0) {
+  if (conflicts.length > 0) {
     console.log(c.redBright('Can\'t alter schema, there are conflicts.'));
```

Comment:
this is due to eslint enforcing single quote, i shall see to give you an updated config as it's  seems better to enforce double quote in that particular case